exercisesResolution/12.classBankAccount.js

```diff
@@ -0,0 +1,44 @@
+class BankAccount {
+  constructor (clientName, accountNumber, balance,){
+    if (balance <= 0) {
+      throw new Error ('The balance must be greater than zero to create a new account!');
+    }
+    this.clientName = clientName;
+    this.accountNumber = accountNumber;
+    this.balance = balance;  
+    this.bankStatement = [];
+  }
+
+  getBankStatement(operation){
```

Comment:
O metodo de adicionar uma nova entrada no vetor de operações (backStatment) está esperando apenas uma operação (seja ela o que for). Isso significa que quem chama o metodo é livre para mandar o que quiser, seja uma opreação, um numero, uma string etc.

Talvez fosse melhor que getBankStatement recebesse (type, value) ou algo assim e ela mesma contruisse a string e salvasse no banco.